scaffolds/cpp/src/module.h

```diff
@@ -0,0 +1,9 @@
+#ifndef MODULE_H_
```

Comment:
it's more common to put the header files into a directory `include`. The reason is that they consist in the facade of the library and can be easily packaged separately.

Reply:
I agree. I've moved the header file to the `include` folder.